qemu/tests/power_htm.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_package
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run htm cases:
+    Case one
+    1) Download unit test suite and configure it
+    2) Run kvm test on host
+    3) Check host is still available
+
+    Case two
+    1) Download test application in the guest
+    2) Run it in the guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    if params["unit_test"] == "yes":
+        error_context.context("Prepare unit test on host", logging.info)
+        cmds = []
+        cmds.append(params["get_htm_dir"]), cmds.append(params["compile_htm"])
+        for cmd in cmds:
```

Comment:
You can just define the `cmds` list with `[params["get_htm_dir"]), cmds.append(params["compile_htm"]]` directly, not need define an empty list at first.

Reply:
Removed

Reply:
Updated.

Reply:
finsihed